### ISSUE TITLE
Switch to serif version of Chinese and Japanese fonts

### ISSUE THREAD

**Issue by thesunshade** [suttacentral/suttacentral#3002]
Currently Chines and Japanese texts are using the Noto Sans versions of their fonts. This is not the idea font for long text reading. 

Using the Noto Serif versions would be an improvement.

Interface text could be set to use any sanserif font the user had on their machine already.

Tasks:

- [ ] subset/optimize the Chinese and Japanese Noto Serif fonts for SuttaCentral
- [ ] change CSS so that these were used for the sutta texts
- [ ] change CSS so that interface items used whatever local sanserif font the user had.

**Comment by thesunshade** [comment]
@ihongda, I have talked with Bhante @sujato about this and he generally thinks it's a good idea. We just need to make sure that users aren't forced to always download two font types (Serif *and* Sans) and that the font we do serve is as small as possible.

**Comment by sujato** [comment]
Yes, this is a good idea, stylistically I'd like to use serif fonts for the texts, but the download size is a real consideration. Let's test it and see how much it ends up.

**Comment by wcang** [comment]
Noto Serif TC is even harder to read at compared to Noto Sans unless I zoom in dramatically.
![image](https://github.com/user-attachments/assets/50203350-31c7-487c-842f-d5cfce2e0f4d)
Compared this to the original Noto Sans CJK TC. Notice the word 月 is a bit blurry.
![Screenshot 2025-01-07 090128](https://github.com/user-attachments/assets/f9b0eec4-ba49-40f4-8a80-77871d37039a)
But I am probably doing something wrong because I'm playing around in Firefox inspector without knowing how CSS and font interacts.

By the way, what is the difference between Noto Sans CJK TC and Noto Sans TC?

**Comment by thesunshade** [comment]
Neither of those look like serif to me. In the case of CJK I think the term "serif" is a bit of a misnomer. We are really talking about serif being variable stroke width.

https://fonts.google.com/noto/specimen/Noto+Sans+TC

https://fonts.google.com/noto/specimen/Noto+Serif+TC

**Comment by wcang** [comment]
@thesunshade You're right. I pasted the Noto sans screenshot twice. My bad!
![Screenshot 2025-01-07 090039](https://github.com/user-attachments/assets/49d10ef3-f4d5-4925-9fbf-2e7ed5aa3465)
Here is the screenshot for Noto Serif TC. Look at the word 月

**Comment by ihongda** [comment]
![Image](https://github.com/user-attachments/assets/fc6400e3-5ad4-4d32-86b1-ce28501afaf5)


This is `Source Han Serif` from Adobe, But the font file size comes to 19.3MB.

**Comment by ihongda** [comment]
![Image](https://github.com/user-attachments/assets/87c35000-5953-433a-b41e-3b0ad76beef1)

For Japanese fonts, I tried using the Japanese government standard font (left).

**Comment by thesunshade** [comment]
Let me pull in the user who made the original request.

**Comment by kaz-kaz** [comment]
Hello everyone, and the Japanese text with the government standard text in the above example looks very suitable for this type of text. I would say it has much more a sophisticated looks. Thank you very much for the efforts and support.

**Comment by ihongda** [comment]
Hi @kaz-kaz 
Thank you, I will try to use this font instead of the current font.

**Comment by ihongda** [comment]
Hi @kaz-kaz @thesunshade 
I have made relevant changes and pushed them to `staging`

https://staging.suttacentral.net/dn1/jpn/kansai?lang=en&reference=none&highlight=false

**Comment by wcang** [comment]
The text on staging looks good. It got rid of the problem highlighted in this thread: https://discourse.suttacentral.net/t/the-right-chinese-font-family/37442

Thank you very much @ihongda
